README.md

```diff
@@ -1,4 +1,109 @@
-# lto-api.java
- LegalThings One client for Java
+LTO Network client for PHP
+===
 
-**WIP - Please [comment on this issue](https://github.com/ltonetwork/lto-api.java/issues/5) if you'd like to connect to LTO Network from Java.**
+_Signing and addresses work for the public chain only._
+
+Requirements
+---
+
+- JDK 11 or later
+
+Accounts
+---
+
+### Creation
+
+#### Create an account from seed
+
+```java com.ltonetwork.client.core
+String seed = "my seed phrase";
+
+AccountFactory af = new AccountFactory("Testnet");
+Account acc = af.seed(seed);
+```
+
+#### Create an account from sign and encrypt public keys
+
+```java com.ltonetwork.client.core
+Key signKey = new Key("mySignKeyValue", Encoding.BASE58);
+Key encryptKey = new Key("myEncryptKeyValue", Encoding.BASE58);
+
+AccountFactory af = new AccountFactory("Testnet");
+Account acc = af.createPublic(signKey, (byte) 84, encryptKey);
```

Comment:
The network char 'T' is passed to the constructor and should be passed to the create functions.